### DISCUSSION TITLE
functions-events to handle function crash

### DISCUSSION THREAD

**Discussion by Maxbrt18** [appwrite/appwrite#4908] [Q&A] [answered]
I wonder if there is an event during the crash of a function to send a mail during it.

By the way, I think the function part of the document (here: `https://appwrite.io/docs/events#functions-events`) is wrong because it talks about buckets.

**Comment by stnguyen90** [comment, marked as answer]
@Maxbrt18 thanks for opening this!

> I wonder if there is an event during the crash of a function to send a mail during it.

Is this the same as https://github.com/appwrite/appwrite/issues/4646? If so, it would be best to 👍🏼 that issue and close this.

> By the way, I think the function part of the document (here: https://appwrite.io/docs/events#functions-events) is wrong because it talks about buckets.

I think this was raised in https://github.com/appwrite/appwrite/issues/4862. Similarly, it would be good to 👍🏼 that issue and close this.
